packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -144,6 +144,13 @@ abstract class FlutterCommand extends Command<void> {
         'will select a random open port on the host.',
       hide: hide,
     );
+    argParser.addFlag('web-skip-launch',
+      defaultsTo: false,
+      help: 'Whether to skip automatically launching browsers on web devices that do so. '
+      'Setting this to true allows using the debug chrome extension in environments that don\'t'
```

Comment:
Maybe worth clarifying this is the "Dart Debug Extension" for those reading the message and wondering what to look for.

Reply:
Done